server/suite.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"sync/atomic"
+
+	"config-etcd/etcd"
+	"github.com/cloudwego/kitex/server"
+)
+
+const (
+	limiterConfigName = "limit"
+)
+
+var num int64
```

Comment:
Better implemented as a standalone package since it will also be used by clients.